src/components/mobile-menu/mobile-menu.module.scss

```diff
@@ -64,12 +64,13 @@
   display: flex;
   align-items: center;
   flex-direction: row;
+  position: static;
 }
 
 .instagramSpace {
-  padding: 0 2rem;
+  margin: 0 2rem;
 }
 
 .gitSpace {
-  padding-left: 2rem;
+  margin-left: 2rem;
```

Comment:
I think you can avoid defining these class using: https://www.w3schools.com/cssref/sel_nth-child.php#:~:text=The%20%3Anth%2Dchild(n,name)%2C%20of%20its%20parent.

Reply:
Done